### PR TITLE
Create pt-BR translation file

### PR DESCRIPTION
**Pull request by cain-dev**
create a different file to Brazilian Portuguese

### CODE REVIEW

**Comment by Torxed** [comment]
It would be nice with a english topic and description.
I get that this is a translation of some sort? :)

**Comment by svartkanin** [comment]
@cain-dev can you please run the script again to also generate tge .mo file, otherwise the translation won't actually work.

In addition I don't know if ptbr is a valid locale abbreviation so it  might not get picked up, so would be good if you could give it a try after generating the .mo file :)

**Comment by Alexmelman88** [comment]
The correct spelling for the Brazilian locale is pt_BR, in other projects it is.

**Comment by Torxed** [comment]
> > @cain-dev can you please run the script again to also generate tge .mo file, otherwise the translation won't actually work.
> > In addition I don't know if ptbr is a valid locale abbreviation so it might not get picked up, so would be good if you could give it a try after generating the .mo file :)
> 
> script? sorry, i have always changed the text of the base file to write the msgtr on nvim. What is the correct way to do this? im gonna create a corret pull request then

You might just be able to run the script with your current changes :)
It's mentioned in the locale README: https://github.com/archlinux/archinstall/blob/master/archinstall/locales/README.md

**Comment by cain-dev** [comment]
i will create a temporary directory just for reference

**Comment by cain-dev** [comment]
could you guys accept the pull just to have some file to someone else who wants to contribute plz?

**Comment by dylanmtaylor** [comment]
> could you guys accept the pull just to have some file to someone else who wants to contribute plz? it's almost finished btw

It would be preferable to not have half done translations in the repo. My vote is to wait until it is ready

**Comment by cain-dev** [comment]
> > could you guys accept the pull just to have some file to someone else who wants to contribute plz? it's almost finished btw
> 
> It would be preferable to not have half done translations in the repo. My vote is to wait until it is ready

it's not half done, it's more than 80% done, also KDE br team always come here to help translate

it's just 1 txt file, nothing more for some hours of my free work, just trying to help cause i have to rest a little

File: archinstall/locales/pt_BR/LC_MESSAGES/tradução.txt [hunk 1]
```diff
@@ -0,0 +1,759 @@
...
+"Use NetworkManager (necessary to configure internet graphically in GNOME and "
+"KDE)"
+msgstr "Usar NetworkManager (necessário para configurar internet graficamente no GNOME e KDE)"
+
+msgid "Select one network interface to configure"
+msgstr "Selecione uma interface de rede para configurar"
+
+msgid ""
+"Select which mode to configure for \"{}\" or skip to use default mode \"{}\""
+msgstr "Selecione qual modo configurar para \"{}\" ou ignore para usar o modo padrão \"{}\"
```
Comment (Alexmelman88) [inline archinstall/locales/pt_BR/LC_MESSAGES/tradução.txt:77]:
```suggestion
msgstr "Selecione qual modo configurar para \"{}\" ou ignore para usar o modo padrão \"{}\""
```

File: archinstall/locales/pt_BR/LC_MESSAGES/tradução.txt [hunk 2]
```diff
@@ -0,0 +1,759 @@
...
+
+msgid "Set desired subvolumes on a btrfs partition"
+msgstr "Definir subvolumes desejados numa partição btrfs"
+
+msgid ""
+"{}\n"
+"\n"
+"Select which partition to set subvolumes on"
+msgstr ""
+{}\n"
```
Comment (Alexmelman88) [inline archinstall/locales/pt_BR/LC_MESSAGES/tradução.txt:486]:
```suggestion
"{}\n"
```

**Comment by Torxed** [comment]
> > > could you guys accept the pull just to have some file to someone else who wants to contribute plz? it's almost finished btw
> > 
> > 
> > It would be preferable to not have half done translations in the repo. My vote is to wait until it is ready
> 
> it's not half done, it's more than 80% done (just me), also KDE br team always come here to help translate
> 
> it's just 1 txt file, nothing more for some hours of my free work, just trying to help cause i have to rest a little
> 
> just trying not lose my done work and not make them lose time, that's it

Understandable, ping me when you are satisfied and feel ready and I'll have a look at it :)

File: archinstall/locales/pt_BR/LC_MESSAGES/tradução.txt [hunk 3]
```diff
@@ -0,0 +1,759 @@
...
+msgstr "Digite o setor final da partição (porcentagem ou número de bloco, ex: {}): "
+
+msgid "{} contains queued partitions, this will remove those, are you sure?"
+msgstr "{} contém partições em fila, isto irá remover essas, tem certeza?"
+
+msgid ""
+"{}\n"
+"\n"
+"Select by index which partitions to delete"
+msgstr "Selecione por índice quais partições remover"
```
Comment (Alexmelman88) [inline archinstall/locales/pt_BR/LC_MESSAGES/tradução.txt:116]:
```suggestion
msgstr ""
"{}\n"
"\n"
"Selecione por índice quais partições remover"
```

File: archinstall/locales/pt_BR/LC_MESSAGES/tradução.txt [hunk 4]
```diff
@@ -0,0 +1,759 @@
...
+
+msgid ""
+"{}\n"
+"\n"
+"Select by index which partition to mount where"
+msgstr "Selecione por índice quais partições montar em"
```
Comment (Alexmelman88) [inline archinstall/locales/pt_BR/LC_MESSAGES/tradução.txt:122]:
```suggestion
msgstr ""
"{}\n"
"\n"
"Selecione por índice quais partições montar em"
```

File: archinstall/locales/pt_BR/LC_MESSAGES/tradução.txt [hunk 5]
```diff
@@ -0,0 +1,759 @@
...
+msgstr " * Os pontos de montagem das partições são relativos aos de dentro da instalação, o boot seria /boot por exemplo."
+
+msgid "Select where to mount partition (leave blank to remove mountpoint): "
+msgstr "Selecione onde montar a partição (deixe em branco para remover o ponto de montagem): "
+
+msgid ""
+"{}\n"
+"\n"
+"Select which partition to mask for formatting"
+msgstr "Selecione qual partição mascarar para formatar"
```
Comment (Alexmelman88) [inline archinstall/locales/pt_BR/LC_MESSAGES/tradução.txt:136]:
```suggestion
msgstr ""
"{}\n"
"\n"
"Selecione qual partição mascarar para formatar"
```

File: archinstall/locales/pt_BR/LC_MESSAGES/tradução.txt [hunk 6]
```diff
@@ -0,0 +1,759 @@
...
+
+msgid ""
+"{}\n"
+"\n"
+"Select which partition to mark as encrypted"
+msgstr "Selecione qual partição marcar como encriptada"
```
Comment (Alexmelman88) [inline archinstall/locales/pt_BR/LC_MESSAGES/tradução.txt:142]:
```suggestion
msgstr ""
"{}\n"
"\n"
"Selecione qual partição marcar como encriptada"
```

File: archinstall/locales/pt_BR/LC_MESSAGES/tradução.txt [hunk 7]
```diff
@@ -0,0 +1,759 @@
...
+
+msgid ""
+"{}\n"
+"\n"
+"Select which partition to mark as bootable"
+msgstr "Selecione qual partição marcar como bootável"
```
Comment (Alexmelman88) [inline archinstall/locales/pt_BR/LC_MESSAGES/tradução.txt:148]:
```suggestion
msgstr ""
"{}\n"
"\n"
"Selecione qual partição marcar como bootável"
```

File: archinstall/locales/pt_BR/LC_MESSAGES/tradução.txt [hunk 8]
```diff
@@ -0,0 +1,759 @@
...
+
+msgid ""
+"{}\n"
+"\n"
+"Select which partition to set a filesystem on"
+msgstr "Selecione qual partição definir um sistema de arquivos"
```
Comment (Alexmelman88) [inline archinstall/locales/pt_BR/LC_MESSAGES/tradução.txt:154]:
```suggestion
msgstr ""
"{}\n"
"\n"
"Selecione qual partição definir um sistema de arquivos"
```

File: archinstall/locales/pt_BR/LC_MESSAGES/tradução.txt [hunk 9]
```diff
@@ -0,0 +1,759 @@
...
+msgid "Select which filesystem your main partition should use"
+msgstr "Selecione qual sistema de arquivos a partição principal deverá usar"
+
+msgid "Current partition layout"
+msgstr "Esquema de partições atual"
+
+msgid ""
+"Select what to do with\n"
+"{}"
+msgstr "Selecione o que fazer com\n"
```
Comment (Alexmelman88) [inline archinstall/locales/pt_BR/LC_MESSAGES/tradução.txt:97]:
```suggestion
msgstr "Selecione o que fazer com\n"
"{}"
```

File: archinstall/locales/pt_BR/LC_MESSAGES/tradução.txt [hunk 10]
```diff
@@ -0,0 +1,759 @@
...
+msgid ""
+"For the best compatibility with your Nvidia hardware, you may want to use "
+"the Nvidia proprietary driver.\n"
+msgstr "Para melhor compatibilidade com o seu hardware Nvidia, talvez queira usar o driver proprietário da Nvidia."
+
+msgid ""
+"\n"
+"\n"
+"Select a graphics driver or leave blank to install all open-source drivers"
+msgstr "Selecione um driver de vídeo ou deixe em branco para instalar todos os drivers open-source"
```
Comment (Alexmelman88) [inline archinstall/locales/pt_BR/LC_MESSAGES/tradução.txt:205]:
```suggestion
msgstr ""
"\n"
"\n"
"Selecione um driver de vídeo ou deixe em branco para instalar todos os drivers open-source"
```

File: archinstall/locales/pt_BR/LC_MESSAGES/tradução.txt [hunk 11]
```diff
@@ -0,0 +1,759 @@
...
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+msgid "[!] A log file has been created here: {} {}"
+msgstr "[!] Um arquivo de log foi criado aqui: {} {}"
+
+msgid ""
+"    Please submit this issue (and file) to https://github.com/archlinux/"
+"archinstall/issues"
+msgstr "Por favor envie este problema (e o arquivo) para https://github.com/archlinux/archinstall/issues"
```
Comment (Alexmelman88) [inline archinstall/locales/pt_BR/LC_MESSAGES/tradução.txt:15]:
```suggestion
msgstr "    Por favor envie este problema (e o arquivo) para https://github.com/archlinux/archinstall/issues"
```

**Review by Alexmelman88 — CHANGES_REQUESTED** [review]

File: archinstall/locales/pt_BR/LC_MESSAGES/tradução.txt (hunk 11 above)
Comment (cain-dev) [reply archinstall/locales/pt_BR/LC_MESSAGES/tradução.txt:15]: thank you

**Comment by cain-dev** [comment]
@Torxed , everything is there, could you guys please just run the script for me and merge? thanks

**Comment by dylanmtaylor** [comment]
> @Torxed , everything is there, could you guys please just run the script for me and merge? thanks

Thanks for the translation work! I'm sure someone else could run the script and make another commit, that shouldn't be an issue.

**Comment by svartkanin** [comment]
It has to be someone with superpowers to commit to this PR :)

**Comment by cain-dev** [comment]
the script isn't working for me, where is the pt_BR on the json file?

**Comment by dylanmtaylor** [comment]
> It has to be someone with superpowers to commit to this PR :)

You can always make a merge request against his branch to add your commits.

**Pull request merged**